mmv1/products/compute/api.yaml

```diff
@@ -14674,6 +14674,36 @@ objects:
               https://cloud.google.com/vpc/docs/flow-logs#filtering for details on how to format this field.
               The default value is 'true', which evaluates to include everything.
             default_value: "true"
+      - !ruby/object:Api::Type::Enum
+        name: 'stackType'
+        update_verb: :PATCH
+        update_url: projects/{{project}}/regions/{{region}}/subnetworks/{{name}}
+        fingerprint_name: 'fingerprint'
+        values:
+          - :IPV4_ONLY
+          - :IPV4_IPV6
+        default_value: :IPV4_ONLY
```

Comment:
I don't think we can apply a default here! The API doesn't return the value for older resources, so it wouldn't be safe to apply.

Here's an example from a couple nights ago:

```
{
  "id": "9043695605119007144",
  "creationTimestamp": "2021-09-23T06:55:19.039-07:00",
  "name": "tf-test-xyvapdt3w1",
  "selfLink": "https://www.googleapis.com/compute/beta/projects/ci-test-project-188019/global/networks/tf-test-xyvapdt3w1",
  "autoCreateSubnetworks": false,
  "routingConfig": {
    "routingMode": "REGIONAL"
  },
  "kind": "compute#network"
}
```

There's a few mitigation options:

1. Remove the default, do nothing else
    * This is probably what I'd do- it has a mix of risk and safety.
2. Remove the default, make the field O+C
    * This is technically the safest option, but has the downside of making the field O+C. It's safest because if the API started returning it out of the blue, Terraform wouldn't have a diff. This is probably not worth applying.
3. Add `default_if_empty: true` in `terraform.yaml`
    * This gives us a Terraform-side default, which is useful information at plan time, but means that if the default ever changes we may handle it _very_ badly. `default_if_empty` rewrites all empty responses into the default, so if empty means something else Terraform'll confuse users.